kotest/src/main/kotlin/com/github/erikhuizinga/mockk/kotest/MockkTestListener.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.erikhuizinga.mockk.kotest
+
+import io.kotlintest.Spec
+import io.kotlintest.TestCase
+import io.kotlintest.TestResult
+import io.kotlintest.extensions.TestListener
+import io.mockk.MockKAnnotations
+import io.mockk.clearAllMocks
+import io.mockk.unmockkAll
+
+/**
+ * [TestListener] for kotest.
+ * [beforeSpec] initializes mocks with annotations.
+ * [afterTest] clears all mocks.
+ * [afterSpec] cancels all mocks.
```

Comment:
Just in case a user uses more than just MockK in their tests, let's specify this is for MockK only.

```suggestion
 * [beforeSpec] initializes MockK mocks with annotations.
 * [afterTest] clears all MockK mocks.
 * [afterSpec] unmocks all MockK mocks.
```